libpod/container_internal_linux.go

```diff
@@ -2377,7 +2377,17 @@ func (c *Container) getHosts() string {
 
 		if depCtr != nil {
 			host := ""
+			if depCtr.config.NetMode.IsSlirp4netns() {
+				// When using slirp4netns, the interface gets a static IP
+				if slirp4netnsIP, err := GetSlirp4netnsIP(depCtr.slirp4netnsSubnet); err != nil {
+					logrus.Warnf("Failed to determine slirp4netnsIP: %v", err.Error())
+				} else {
+					return fmt.Sprintf("%s\t%s %s\n", slirp4netnsIP.String(), c.Hostname(), c.config.Name)
+				}
+			}
```

Comment:
This block has to be moved to line 2318, you are changing the host.containers.internal code here.

Reply:
I did not find the depCtr until now.